intelmq/bots/BOTS

```diff
@@ -728,6 +728,18 @@
                 "port": "<port>"
             }
         },
+        "Redis": {
+            "description": "Redis is the bot responsible to send events to a Redis server.",
+            "module": "intelmq.bots.outputs.redis.output",
+            "parameters": {
+                "redis_server_ip": "127.0.0.1",
+                "redis_server_port": "6379",
+                "redis_db": "2",
+                "redis_queue": "external-redis-queue",
+                "redis_timeout": "50000",
```

Comment:
Without a pushed commit I can't check the whitespaces. But 'E' < 'e' (Ascii ;) )